File: Stage12/Java/Problem1436.java
package Stage12.Java;

import java.util.Scanner;

public class Problem1436 {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);

        int N = sc.nextInt();
        int num = 666;
        int count = 0;

        while (true) {
            if (String.valueOf(num).contains("666")) {
                count++;
            }

            if (count == N) {
                System.out.println(num);
                break;
            }
            num++;
        }
        
        sc.close();
    }
}

File: Stage12/C/Problem1436.c
#include <stdio.h>
#include <string.h>

int main(void) {

    int N;
    scanf("%d", &N);

    int num = 666;
    int count = 0;
    char str[20];

    while (1) {
        sprintf(str, "%d", num);

        if (strstr(str, "666") != NULL) {
            count++;
        }

        if (count == N) {
            printf("%d", num);
            break;
        }

        num++;
    }

    return 0;
}